portals/publisher/src/main/webapp/source/src/app/data/AuthManager.js

```diff
@@ -206,31 +206,27 @@ class AuthManager {
     static isRestricted(scopesAllowedToEdit, api = {}) {
         // determines whether the apiType is API PRODUCT and user has publisher role, then allow access.
         if (api.apiType === 'APIPRODUCT') {
-            if (AuthManager.getUser().scopes.includes('apim:api_publish')) {
-                return false;
-            } else {
-                return true;
-            }
+            return !user.scopes.includes('apim:api_publish');
```

Comment:
@GavinHemsada the `user` object here `return !user.scopes.includes('apim:api_publish');` does not seem to have been initialised.